java/runtime/src/main/java/org/ray/runtime/RuntimeContextImpl.java

```diff
@@ -0,0 +1,54 @@
+package org.ray.runtime;
+
+import com.google.common.base.Preconditions;
+import org.ray.api.RuntimeContext;
+import org.ray.api.id.UniqueId;
+import org.ray.runtime.config.RunMode;
+import org.ray.runtime.config.WorkerMode;
+
+public class RuntimeContextImpl implements RuntimeContext {
+
+  private AbstractRayRuntime runtime;
+
+  public RuntimeContextImpl(AbstractRayRuntime runtime) {
+    this.runtime = runtime;
+  }
+
+  @Override
+  public UniqueId getCurrentDriverId() {
+    return runtime.getWorkerContext().getCurrentDriverId();
+  }
+
+  @Override
+  public UniqueId getCurrentActorId() {
+    Preconditions.checkState(runtime.rayConfig.workerMode == WorkerMode.WORKER);
+    return runtime.getWorker().getCurrentActorId();
+  }
+
+  @Override
+  public boolean wasCurrentActorReconstructed() {
+    Preconditions.checkState(
+        runtime.getWorkerContext().getCurrentTask().isActorCreationTask());
```

Comment:
1) also need to check `runtime.getWorkerContext().getCurrentTask() != null`.
2) print error message `This method can only be called from an actor creation task.`